cmd/podman/common.go

```diff
@@ -242,22 +242,42 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 	createFlags.Bool(
 		"help", false, "",
 	)
+	createFlags.String(
+		"health-cmd", "",
+		"set a healthcheck command for the container ('none' disables the existing healthcheck)",
+	)
 	createFlags.String(
 		"healthcheck-command", "",
```

Comment:
I think Cobra allows aliases for flags - it'd be better to add these as aliases to the existing flags, not entirely new ones

Reply:
Can you point me to some docs, please? I've looked but could not find anything. This is why I've followed the same pattern that was used for the `--net`, `--network` pair.

(Cobra allows shorthands, as in `--publish, -p` with `StringP`, but that's not what is needed in this case.)

Reply:
I'm almost certain we used this somewhere... Let me poke around

Reply:
I might have been thinking of urfave/cli instead, which AFAIK allowed arbitrary numbers of flag aliases. Which, in cases like this, was a good bit more convenient...